scripts/install_secp.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+
+set -e
+
+cd crypto
+git clone https://github.com/bitcoin-core/secp256k1
+cd secp256k1
+./autogen.sh
+./configure --enable-jni --enable-experimental --enable-module-schnorr --enable-module-ecdh --prefix=$PWD/.tmp
+make install
```

Comment:
`make` is probably sufficient.

Reply:
oh, right, with `make` without `install` we wouldn't need the `--prefix`, right?